src/test-data/multipart-response-test-data.js

```diff
@@ -0,0 +1,170 @@
+// Invalid case where multi-part message has just one part. This should never happen in real world.
+{
+  const rawData = String.raw`--------abcde123
+Content-Type: application/json; charset=UTF-8
```

Comment:
Please maintain indentation here. The dataset of constants is confusing.

Reply:
Unfortunately, it can't be done. The multi-part format won't tolerate unnecessary spaces.